certs/client.crt

```diff
@@ -0,0 +1,13 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We only need the CA cert to validate the connection. The data provider can update their cert or issue many for different machines as long as they are from the same CA.